src/Microsoft.AspNetCore.AzureAppServices.TestBundle/Microsoft.AspNetCore.AzureAppServices.TestBundle.csproj

```diff
@@ -22,6 +22,7 @@
   <ItemGroup>
     <Content Include="applicationHost.xdt" />
     <Content Include="install.cmd" />
+    <Content Include="dotnet.cmd" />
```

Comment:
is this required (as opposed to setting the path?)

Reply:
We can not set path for git deploy, and directory I'm copying this to is first in PATH so everything fit's nicely.